app/client/src/routes.js

```diff
@@ -14,8 +14,8 @@ angular.module('reg')
     // Set up de states
     $stateProvider
       .state('login', {
-        url: "/login",
-        templateUrl: "views/login/login.html",
+        url: "/home",
+        templateUrl: "views/home/home.html",
```

Comment:
We should try to not modify the states set up by these express routes. Login should be routing to the login view and nothing else.